diff -Naur oneVPL-intel-gpu-intel-onevpl-22.4.2.old/_studio/shared/mfx_logging/include/mfx_utils_logging.h oneVPL-intel-gpu-intel-onevpl-22.4.2/_studio/shared/mfx_logging/include/mfx_utils_logging.h
--- oneVPL-intel-gpu-intel-onevpl-22.4.2.old/_studio/shared/mfx_logging/include/mfx_utils_logging.h	2022-05-25 15:28:07.919706923 +0200
+++ oneVPL-intel-gpu-intel-onevpl-22.4.2/_studio/shared/mfx_logging/include/mfx_utils_logging.h	2022-05-25 15:28:11.690776580 +0200
@@ -30,6 +30,7 @@
 #include <string>
 #include <map>
 #include <sstream>
+#include <memory>
 
 typedef enum
 {
